### DISCUSSION TITLE
Game mode similar to Project Diva for controller users

### DISCUSSION THREAD

**Discussion by Kissadere** [ppy/osu#18481] [Ideas] [answered]
Hello!

I'd like to make a suggestion. **A new gamemode** featuring the gameplay from Hatsune Miku: Project Diva.
Where you use your controller at the rythm of the song.
Heres an example of what i'm refering: https://www.youtube.com/watch?v=1AQ4pow_Kvs

After doing some research, i found out no one has suggested this. Maybe is not possible due to copyright issues, i don't know if that specific gamemode/gameplay is patented or something.

**Comment by bdach** [comment]
developers can already extend lazer with their own rulesets (https://github.com/ppy/osu/discussions/13096). if someone is willing to develop this on their own time the tools are all there.

**Comment by Theighlin** [comment, marked as answer]
I think this is what you're looking for: https://github.com/Artemis-chan/osu-DIVA

Though I don't know how or if it works with a controller

**Reply by Kissadere** [reply]
Just tried it, works terrible.

**Reply by bdach** [reply]
you can write a better one if you're so inclined (or help with that one). the world is your oyster.
